Makefile

```diff
@@ -344,12 +344,15 @@ markdownLintImage=tmknom/markdownlint
 containerMarkdownLint=$(PROJECT_NAME)-markdownlint
 containerMarkdownLintFix=$(PROJECT_NAME)-markdownlint-fix
 
+golangci_lint_cmd=go run github.com/golangci/golangci-lint/cmd/golangci-lint
+
+# NOTE: if you change the `lint` job then you must update ./github/workflows/lint.yml
 lint:
-	golangci-lint run --out-format=tab
+	$(golangci_lint_cmd) run --out-format=tab
 	@if docker ps -a --format '{{.Names}}' | grep -Eq "^${containerMarkdownLint}$$"; then docker start -a $(containerMarkdownLint); else docker run --name $(containerMarkdownLint) -i -v "$(CURDIR):/work" $(markdownLintImage); fi
```

Comment:
Why do we have docker commands here?

Reply:
This is for Markdown linting, not for Go code. We use a Tendermint container image here.